modules/dns-response-policy/README.md

```diff
@@ -0,0 +1,92 @@
+# Google Cloud DNS Response Policy
+
+This module allows management of a [Google Cloud DNS policy and its rules](https://cloud.google.com/dns/docs/zones/manage-response-policies). The policy can be already existing and passed in via its id.
```

Comment:
Feel free to ignore :)

The module allows managing a [Google Cloud DNS policy and its rules](https://cloud.google.com/dns/docs/zones/manage-response-policies). The policy can already exist and passed in via its id.